java/src/org/openqa/selenium/grid/commands/InfoCommand.java

```diff
@@ -143,11 +146,11 @@ private String readContent(String path) throws IOException {
         } else if ("```".equals(line)) {
           inCode = !inCode;
         } else {
-          if (line.startsWith("=")) {
+          if (line.charAt(0) == '=') {
```

Comment:
Any reason for modifying this?

Reply:
Minor performance improvement, while I'm in the same class. The `#startsWith(String)` method can take a String of multiple characters, so it needs to search through the String. Whereas if you're only searching for a single character, `charAt(int)` directly accesses the char[] backing the String and returns the value in a single call.

Not a `Pattern`, so I can move it into another PR if you think that's appropriate.